docs/configuration.md

```diff
@@ -14,107 +12,3 @@ Check the documentation for
 - [plugins](./plugins.md)
 
 to learn more about different types of configuration and features available.
-
-## Migration guide
```

Comment:
I think it would be nice of ours to leave this section and link to the certain github tag, so it's still easy to find. I'd assume there's still a lot of folks using RN 0.59 out there.

Reply:
I love this idea! I'll just link to the version tag with a note that maybe `rnpm` config is only supported within React Native <= 0.61 - in that case, check the migration guides under ...?